definitions-terms.md

```diff
@@ -0,0 +1,108 @@
+* [Registry](#registry)
+* [SBoM](#sbom)
+* [Subject](#subject)
+* [Tag](#tag)
+
+## Artifact / Image
+
+Artifacts are a generalization of how images are stored in an instance of the [OCI Distribution Spec][oci-distribution-spec]. An [OCI Image][oci-image] is a type of [OCI Artifact][oci-artifact]. When referencing artifacts, within a [fully qualified reference](#registry-fully-qualified-reference), the artifact is right most element of the namespace, combined with the `:tag`.  
+Reference: [OCI Artifacts][oci-artifact]  
+![](./media/artifact-ref.png)  
+
+
+## Fingerprint
+
+A short identifier of a given public key.  
+Reference: [Fingerprint][fingerprint]
+
+## Multi-Tenant Registry
+
+A collection of unique [registries](#registry), stored within an instance of an [OCI Distribution Spec][oci-distribution-spec]. Registry operators use one of two methods to uniquely identify an org. In either case, information within that org is considered unique to the org, including all key references and authentication.
+
+## OCI Annotations
+
+A key-value map that can be associated with [OCI Descriptors][oci-descriptor] and [OCI Image manifests][oci-manifest].  
+OCI spec reference: [OCI Annotations][oci-annotations]
+
+## OCI Descriptor
+
+A structure describing content, including the media type, a content-addressable digest, the size, and other properties. Descriptors are used to describe layers and configuration in a manifest.  
+OCI spec reference: [OCI Descriptor][oci-descriptor]
+
+## OCI Index
+
+A higher-level collection of [image manifests][oci-manifest], or other [oci indexes][oci-index], typically used to describe platform-specific (architecture and operating-system) images that can be identified collectively and referred to together. The specific image manifests are identified by modified [descriptors][oci-descriptor] with additional properties and restrictions. [OCI Artifacts][oci-artifact] takes this a step further and provides for an OCI Index to reference a collection of artifacts, such as an image, a deployment chart, its SBoM and deployment configuration references.  
+OCI spec reference: [OCI Image Index][oci-index]
+
+## OCI Manifest
+
+A description of an [artifact][oci-artifact] within an instance of an [OCI Distribution][oci-distribution-spec] complaint registry. The manifest references optional configuration and blobs (layers) as content-addressable references ([descriptors][oci-descriptor]).  Artifacts may be container images, helm charts or other artifact types that may be signed and stored in a registry.    
+OCI spec reference: [OCI Image Manifest][oci-manifest]
+
+## Registry
+
+A registry is a collection of [OCI Indexes][oci-index] and [OCI Manifestes][oci-manifest] for a specific org or entity.  
+A unique registry may be uniquely identified with a domain  
+![](./media/unique-registry-domain-ref.png)  
+ Or a root namespace:  
+![](./media/unique-registry-namespace-ref.png)
+
+
+## Registry Namespace
```

Comment:
I'd remove this as namespaces are not a specific part of the definition of a name.  You could include a definition of a "path component", since that's the term used in the distribution spec.

Reply:
Are you suggesting we use the term path, instead of namespace? That makes sense. Although, I'd still want to separate the element that doesn't include the artifact name. 
Here's the larger post that goes into more detail:  
[Registry Names, Namespaces, Images, Artifacts & Tags](https://stevelasker.blog/2020/02/17/registry-namespace-repo-names/)

Reply:
"path component" is the terminology used in the distribution spec, so I would bias toward that.  I don't really see that we need to describe all of the components of a name, though.  It's not clear to me that we need a distinction between "namespaces" and "images", for example.  One of the goals of this project is for the signatures to be independent of the location of the images/artifacts, and right now the name is synonymous with the location.

Reply:
A good conversation for Monday as well.
The issue is what I describe here: https://stevelasker.blog/2020/02/17/registry-namespace-repo-names/
> Potential life cycle of the web:v1.0 artifact. In the following example, the same web:v1.0 artifact is copied across several repos, using different namespaces, as it makes it way through production, to an archived state.

1. org.example.com/**dev/blue-team**/web:v1.0
1. org.example.com/**staging/marketing**/web:v1.0
1. org.example.com/**prod/marketing**/web:v1.0
1. org.example.com/**archive/marketing**/web:v1.0

We need a way for the `web:1.0` artifact to move between repos, while maintaining the same signature.
We've already discussed moving across registries:
org-archive.example.com/**marketing**/web:v1.0

What I'd like to accomplish is defining the path as a separate element of the artifact. An artifact can move between paths without changing.